egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1b.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+# Copyright 2017 University of Chinese Academy of Sciences (UCAS) Gaofeng Cheng
+# Apache 2.0
+
+# Using string-style chunk width, batchnorm in TDNN layers and perframe dropout for LSTMP.
```

Comment:
please make sure the chain_dir_info.pl output is put here, I want to be able to see the number of parameters.